src/main/java/nus/climods/model/AcadYearModuleList.java

```diff
@@ -0,0 +1,44 @@
+package nus.climods.model;
```

Comment:
Put this under the `nus.climods.model.module` package? This shouldn't be on the same level as `ModelManager`. Same for `ReadOnlyAcadYearModuleList`

Reply:
Yes, an oversight by me. Thanks for pointing it out!